components/LeftNavbar.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="border-r border-secondary-bg/40 hover:shadow-inner hover:shadow-secondary-bg/50 transition-shadow">
+    <div>
```

Comment:
I removed this class since it seemed unnecessary? 
We have all the hover effects applied on the individual doctors items.
After I removed it I tested if it made visual a difference on different sizes and I could not see any difference with and without the class. 
Please let me know if there was actually a need for this class.

Reply:
It looks very different on my screen now as the sidebar is bleeding into the tapnav. You'll notice it even more in a dark theme too. 
You could make the shadows stick instead of transitioning if you want.  

I would keep the border alone shadows as they help distinguish the layers between component me which are inspired after googles material design standards for layering.